CMakeLists.txt

```diff
@@ -550,6 +550,7 @@ if(CONFIG_GEN_ISR_TABLES)
     COMMAND ${PYTHON_EXECUTABLE}
     $ENV{ZEPHYR_BASE}/arch/common/gen_isr_tables.py
     --output-source isr_tables.c
+    --kernel zephyr_prebuilt.elf
```

Comment:
To be consistent with the above command this should be:

--kernel $<TARGET_FILE:zephyr_prebuilt>

Reply:
ok